CHANGELOG.md

```diff
@@ -3,13 +3,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 # Upcoming
 
-### 🔄 Changed
+## StreamChat
+### ✅ Added
+- Add support for user blocking [#3223](https://github.com/GetStream/stream-chat-swift/pull/3223)
 
 # [4.57.0](https://github.com/GetStream/stream-chat-swift/releases/tag/4.57.0)
 _June 06, 2024_
 
-## StreamChat
-### ✅ Added
```

Comment:
⚠️ We are removing this and adding "Changed". Please revert this 👍

Reply:
Oops, updated 👍